xl_auth/templates/nav.html

```diff
@@ -22,22 +22,21 @@
         <!-- Collect the nav links, forms, and other content for toggling -->
         <div class="collapse navbar-collapse navbar-ex1-collapse">
             <ul class="nav navbar-nav">
-                <li><a href="{{ url_for('collection.home') }}">{{ _('Collections') }}</a></li>
+                <li><a class="navbar-link" href="{{ url_for('collection.home') }}">{{ _('Collections') }}</a></li>
                 {% if current_user.is_admin %}
-                <li><a href="{{ url_for('user.home') }}">{{ _('Users') }}</a></li>
-                <li><a href="{{ url_for('permission.home') }}">{{ _('Permissions') }}</a></li>
+                <li><a class="navbar-link" href="{{ url_for('user.home') }}">{{ _('Users') }}</a></li>
+                <li><a class="navbar-link" href="{{ url_for('permission.home') }}">{{ _('Permissions') }}</a></li>
                 {% endif %}
             </ul>
             <ul class="nav navbar-nav navbar-right">
                 <li>
-                    <p class="navbar-text">
-                        <a class="navbar-link" href="{{ url_for('user.profile') }}">
-                            {{ _('Logged in as %(user)s', user=current_user.full_name) }}
-                        </a>
-                    </p>
+                    <a class="navbar-link" href="{{ url_for('user.profile') }}">
+                        <img class="navbar-user-gravatar" src="{{ current_user.get_gravatar_url(32) }}" />
```

Comment:
"32" is redundant, since you set that as the default in the `get_gravatar_url` method. Feel free to keep it though, if you think it adds value. Just mentioning since you're new to Python! 😸